core/node/rpc/add_event.go

```diff
@@ -62,6 +63,7 @@ func (s *Service) addParsedEvent(
 	if err != nil {
 		return err
 	}
+	_, _ = s.scrubTaskProcessor.TryScheduleScrub(ctx, streamId)
```

Comment:
Is this the right moment to schedule a scrub task? Should it not be done only after the add succeeded?